pkg/build/controller/strategy/util.go

```diff
@@ -463,10 +463,27 @@ func setupBuildCAs(build *buildv1.Build, pod *corev1.Pod, includeAdditionalCA bo
 				Name: buildapihelpers.GetBuildCAConfigMapName(build),
 			},
 		}
-		if includeAdditionalCA {
+		// Mount the service signing CA key for the internal registry.
+		// This will be injected into the referenced ConfigMap via the openshift/service-ca-operator, and block
+		// creation of the build pod until it exists.
+		//
+		// See https://github.com/openshift/service-ca-operator
+		if includeServiceCA {
 			cmSource.Items = append(cmSource.Items, corev1.KeyToPath{
-				Key:  buildutil.AdditionalTrustedCAKey,
-				Path: buildutil.AdditionalTrustedCAKey,
+				Key:  buildutil.ServiceCAKey,
+				Path: fmt.Sprintf("certs.d/%s/ca.crt", internalRegistryHost),
```

Comment:
@bparees I believe this is what we want to do - use the service signer CA as the CA for the internal registry domain?

Reply:
yes, looks right to me.